src/aqua/remove_worker.aqua

```diff
@@ -0,0 +1,118 @@
+module RemoveWorker declares remove
+export remove, inspect, joined_deals, get_logs, test_join, test, remove_all
+
+import Worker from "@fluencelabs/aqua-lib/workers.aqua"
+import Srv, Op from "@fluencelabs/aqua-lib/builtin.aqua"
+import Spell, StringListValue from "@fluencelabs/spell/spell_service.aqua"
+
+import getWorkersInfo, GetWorkersInfoHostsDeciderInstallationSpells from "workers.aqua"
+import Json from "fluence/peer.aqua"
+import JoinedDeal from "decider/deal_storage.aqua"
+
+use "debug.aqua" as Debug
+use "decider/deal_storage.aqua" as DealStorage
+use "decider/mailbox.aqua" as Mailbox
+
+func spells() -> []GetWorkersInfoHostsDeciderInstallationSpells:
+    info <- getWorkersInfo()
+    decider = info.hosts.decider!
+    <- decider.installationSpells
+
+func remove() -> string:
+    for spell <- spells():
+        on spell.hostId:
+            Worker.remove(spell.workerId)
+    <- "OK"
+
+func joined_deals() -> [][]JoinedDeal:
+    joined: *[]JoinedDeal
+    for spell <- spells():
+        Debug.log_back(["will go to", spell.workerId, spell.hostId])
+        on spell.workerId via spell.hostId:
+            joined <- DealStorage.get_joined_deals("decider")
+            Debug.log_back("")
+    <- joined
+
+func remove_all() -> string:
+    for spell <- spells():
+        on spell.workerId via spell.hostId:
+            spell_id = "decider"
+            joined <- DealStorage.get_joined_deals(spell_id)
+            for deal <- joined:
+                co Debug.log_back(["will remove worker from host", deal.worker_id, spell.hostId])
+                Mailbox.push_remove_worker(spell_id, spell.hostId, deal.worker_id)
```

Comment:
imho, it can be `Worker.request_remove(...)`

Reply:
That's kinda a test api

Reply:
and there's no option to "decline", so it's not a request, it's an order